### ISSUE TITLE
Example Sets: Update Example set with a new question

### ISSUE THREAD

**Issue by sidmundhra** [benetech/MathShare#1430]
Need to add a problem to the example sets currently shown on the main example problem sets (https://mathshare.benetech.org/#/app/problemSet/solve/VV45MTKEMM2LW).

Link to the problem: https://mathshare.benetech.org/#/app/problemSet/edit/WE5CVVTDOQFFS/0

Please make this question no. 2 in the above set.

Problem: 5x+3.

Prompt: Solve for X.

**Comment by sidmundhra** [comment]
@alexrcabral - am I doing this correctly? Amaya needs this actioned immediately.. Can you help me how to flag this one so it can be bumped up?

**Comment by alexrcabral** [comment]
New tasks are go on the product board -> "Ready for Development" column. If something is particularly urgent (e.g. hotfixes) or minimally disruptive content changes like this, we can also flag to John (either @s or through Slack)

**Comment by alexrcabral** [comment]
Also, the full set needs to be complete on the scoping side before it can be surfaced, since that's a content piece. You can do this pretty easily by duplicating the set, adding a problem, reordering it where you'd like it, and surfacing the link to the full, updated set to Rupesh

**Comment by sidmundhra** [comment]
@alexrcabral - Here's the link to the updated set: https://mathshare.benetech.org/#/app/problemSet/view/QPAF6RINGJB7O

@rupeshparab and @johnhbenetech - This need urgent attention as informed to me by Amaya. Please let me know when we can get this live as our main problem set?

Thanks.

**Comment by johnhbenetech** [comment]
@sidmundhra So, to be clear you want us to replace the current example problem set 'Solve for X' with the one linked above?

**Comment by sidmundhra** [comment]
@johnhbenetech: Yes, that's correct. Ideally, I wanted to add just a problem to the existing set but as @alexrcabral pointed out that to reorder the questions I need to duplicate the set and add the question in its appropriate place, so I've done just that with the added question.

**Comment by johnhbenetech** [comment]
@sidmundhra complete
